### ISSUE TITLE
Get Conversations doesn't work

### ISSUE THREAD

**Issue by dgancho** [MicrosoftDocs/bot-docs#1363]
GET /v3/conversations returns (405) MethodNotAllowed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c36ddb50-9e44-96e3-2df3-174166efa75d
* Version Independent ID: 68ac4b64-6616-2216-3519-e0b33d3a53e4
* Content: [API reference - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#get-conversations)
* Content Source: [articles/rest-api/bot-framework-rest-connector-api-reference.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/rest-api/bot-framework-rest-connector-api-reference.md)
* Service: **bot-service**
* GitHub Login: @ivorb
* Microsoft Alias: **kamrani**

**Comment by JonathanFingold** [comment]
@dgancho Closing this as a duplicate of #1357. Please comment on that thread, and we'll try to get you unblocked.
